project/make/validate-spaces

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+
+source "$(dirname "$BASH_SOURCE")/.validate"
+
+#Ignoring files from vendor, ending with .md and .go"
```

Comment:
I'm not sure this comment really adds much -- this seems really apparent to me from the regex itself. :smile: